locust/main.py

```diff
@@ -327,17 +351,19 @@ def kill_workers(children):
             if soft_limit < minimum_open_file_limit:
                 # Increasing the limit to 10000 within a running process should work on at least MacOS.
                 # It does not work on all OS:es, but we should be no worse off for trying.
-                resource.setrlimit(resource.RLIMIT_NOFILE, [minimum_open_file_limit, hard_limit])
+                limits = minimum_open_file_limit, hard_limit
+                resource.setrlimit(resource.RLIMIT_NOFILE, limits)
```

Comment:
not sure this is an improvement?

Reply:
This part will be refactored later anyway. At this point this line changed by mypy request, see failed pipelines earlier.